SoftLayer/CLI/block/list.py

```diff
@@ -0,0 +1,63 @@
+"""List block storage volumes."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import columns as column_helper
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+COLUMNS = [
+    column_helper.Column('id', ('id',)),
+    column_helper.Column('username', ('username',)),
+    column_helper.Column('datacenter',
+                         ('serviceResource', 'datacenter', 'name')),
+    column_helper.Column('storageType', ('storageType', 'keyName')),
+    column_helper.Column('capacityGb', ('capacityGb',)),
+    column_helper.Column('bytesUsed', ('bytesUsed',)),
+    column_helper.Column('ipAddr', ('serviceResourceBackendIpAddress',)),
+]
+
+DEFAULT_COLUMNS = [
+    'id',
+    'username',
+    'datacenter',
+    'storageType',
+    'capacityGb',
+    'bytesUsed',
+    'ipAddr'
+]
+
+
+@click.command()
+@click.option('--username', '-u', help='Volume username')
+@click.option('--datacenter', '-d', help='Datacenter shortname')
+@click.option('--storage_type',
```

Comment:
For the CLI, we use dashes "-" as a word-delimiter because it's slightly easier to type (underscores require hitting shift). Regardless of the reason, we wanted to keep this consistent across all options.